CONTRIBUTING.md

```diff
@@ -1,238 +1,191 @@
+`ember-data` participates in the [RFC process (GitHub emberjs/rfcs)](https://github.com/emberjs/rfcs/).
+Most changes to the public API including new features, changes in behavior, or deprecations require
+community discussion and must go through this process.
 
-## Requesting a Feature
-1. Ember and Ember Data have an RFC process for feature requests. To begin the discussion either
-[gather feedback](https://github.com/emberjs/rfcs/blob/master/README.md#gathering-feedback-before-submitting)
-on the emberjs/rfcs repository. Or, draft an [Ember Data RFC](https://github.com/emberjs/rfcs/pulls?q=is%3Apr+is%3Aopen+label%3Aember-data)
-   - Use RFC pull request for well formed ideas.
-   - Use the `ember-data` label on it.
-   - Use RFC issues to propose a rough idea, basically a great place to test
-     the waters.
+While there is no guarantee that an RFC will be accepted, successful RFCs typically follow a pattern
+of iteration while gathering requirements, addressing feedback, and consensus building. The best RFCs
+are narrowly scoped with clear understanding of alternatives, drawbacks, and their effect on the community.
 
-2. Provide a clear and detailed explanation of the feature you want and why
-it's important to add. Keep in mind that we want features that will be useful
-to the majority of our users and not just a small subset. If you're just
-targeting a minority of users, consider writing an add-on library for Ember.
+    Here are a few suggestions of **steps to take before drafting your RFC** to best make your RFC successful.
+    Often this process will complete quickly, but when it does not, don't despair! Often the best ideas
+    take the longest to bake.
 
-3. If the feature is complex, consider writing an Ember RFC document. If we do
-end up accepting the feature, the RFC provides the needed documentation for
-contributors to develop the feature according the specification accepted by the core team.
+1. Bring up your idea in the `#dev-ember-data` channel on [Discord](https://discord.gg/zT3asNS) or
+   with individual [team members](https://emberjs.com/team/)
+2. Reflect on any concerns, alternatives, or questions that arise from these discussions.
+3. Continue to discuss the idea, giving time for everyone to digest and think about it.
+4. Attend the weekly team meeting to discuss your idea
+5. Open an [RFC issue](https://github.com/emberjs/rfcs/issues?q=is%3Aissue+is%3Aopen+label%3AT-ember-data)
+   to broaden and record the discussion if the idea needs more time for discussion and iteration.
+   * announce your issue in `#dev-ember-data` and anywhere else desired such as `#news-and-announcements` and `twitter`.
+6. [Draft an RFC](https://github.com/emberjs/rfcs#what-the-process-is) and share it with those you have
+   been discussing the ideas with.
+7. Publish your RFC by opening a PR to [emberjs/rfcs/](https://github.com/emberjs/rfcs/pulls?q=is%3Apr+is%3Aopen+label%3AT-ember-data)
+   * label your PR with `T-ember-data`
```

Comment:
I'd add: "(or ask someone in `#dev-ember-data` to add the label if you lack the permission)"